### PR TITLE
Fix mistake in previous fix to `build-cudf-java`

### PR DESCRIPTION
**Pull request by harrism**
My previous fix got the path wrong...

### CODE REVIEW

File: etc/bash-utils.sh [hunk 1]
```diff
@@ -339,7 +339,7 @@ build-cudf-java() {
             ${D_CMAKE_ARGS} \
             -Dmaven.test.skip=true \
             -DCMAKE_EXPORT_COMPILE_COMMANDS=ON \
-            -DCUDF_CPP_BUILD_DIR="$CUDF_HOME/$CUDF_CPP_BUILD_DIR" \
+            -DCUDF_CPP_BUILD_DIR="$CUDF_CPP_BUILD_DIR" \
```
Comment (trxcllnt) [inline etc/bash-utils.sh:342]: Does this need to be a relative path? If absolute is OK, this could be
```
-DCUDF_CPP_BUILD_DIR="$CUDF_ROOT"
```

Comment (harrism) [reply etc/bash-utils.sh:342]: I don't know, but it needs to evaluate to `$CUDF_ROOT/cpp/build/release`, which is what the function that sets `CUDF_CPP_BUILD_DIR` finds, I think.

Comment (trxcllnt) [reply etc/bash-utils.sh:342]: `$CUDF_ROOT` will be the path to the current build dir:
```
$ echo $CUDF_ROOT
/home/ptaylor/dev/rapids/cudf/cpp/build/release
```

But this change looks OK too.

Comment (harrism) [reply etc/bash-utils.sh:342]: I did not know that!

**Pull request merged**